connect/README.md

```diff
@@ -0,0 +1,259 @@
+<h1 align="center">⚡️ hyper connect workshop ⚡️</h1>
+<p align="center">In this workshop, we will dive deeper into hyper-connect and learn some more Data service methods</p>
+
+---
+
+## Table of Contents
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+- [What is hyper-connect?](#what-is-hyper-connect?)
+- [Setup](#setup)
+- [Update a character](#update-a-character)
+- [Remove a character](#remove-a-character)
+- [List characters](#list-characters)
+- [Summary](#summary)
+
+---
+
+## Introduction 
+
+👋🏻 Greetings, Thank you for taking the time to learn hyper with us, in this workshop, we
```

Comment:
Greetings. Thank you for taking the time to learn hyper with us.  In this workshop, we will dive a little deeper into `hyper-connect`,  the deno/nodejs hyper client. But first, if you have not taken the Hello World workshop, please consider taking that workshop first. We will be building on the concepts we learned in previous workshops.